pipeline/inputs/kubernetes-events.md

```diff
@@ -28,8 +28,15 @@ Kubernetes exports it events through the API server. This input plugin allows to
 | tls.verify          | Enable or disable verification of TLS peer certificate.                               | On                                                   |
 | tls.vhost           | Set optional TLS virtual host.                                                        |                                                      |
 
+
+- _* as of Fluent-Bit 3.1 this plugin uses a kubernetes watch stream instead of polling. The interval parameters in versions > 3.1 are used for reconnecting the kubernetes watch stream._
+
+
 ## Getting Started
 
+### Kubernetes Service Account
+The kubernetes service account used by fluent-bit must have `get`, `list`, and `watch` permissions to `namespaces` and `pods` for the namespaces watched in the `kube_namespace` config parameter. If you are using the helm chart to set up fluent-bit, this role is included.
+
```

Comment:
```suggestion
The Kubernetes service account used by Fluent Bit must have `get`, `list`, and `watch`
permissions to `namespaces` and `pods` for the namespaces watched in the
`kube_namespace` configuration parameter. If you're using the helm chart to configure
Fluent Bit, this role is included.
```

Reply:
Both Kubernetes and Fluent Bit are product names, so we capitalize them consistently.